src/PKSim.Core/Snapshots/Mappers/ProjectMapper.cs

```diff
@@ -262,7 +262,7 @@ private void addQualificationPlanToProject(ModelProject project, Model.Qualifica
          project.GetOrCreateClassifiableFor<TClassifiableWrapper, TSubject>(subject);
       }
 
-      private async Task<IEnumerable<Model.Simulation>> allSimulationsFrom(ModelProject project, ProjectContext projectContext, Simulation[] snapshots, SnapshotContext snapshotContext)
+      private async Task<IReadOnlyList<Model.Simulation>> allSimulationsFrom(ProjectContext projectContext, Simulation[] snapshots, SnapshotContext snapshotContext)
```

Comment:
@Yuri05 This returns am empty list of a simulation cannot be loaded. I am wondering why we are not throwing at the end...

Reply:
looks good for me AS IS.
Why would we throw an exception?

Reply:
Well this is how I lost two hours. No error is thrown => run returns success => qualification runner says everything is good > reporting engine barfs with some weird progress bar error

I only found the problem because i started the run from PK-Sim itself  with breakpoints